src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const rentCar = 40;
+
+  if (days >= 7) {
```

Comment:
move all `magic` number to variables

Reply:
isn't fixed. You should move all `magic` numbers to variables
https://www.inf.unibz.it/~calvanese/teaching/04-05-ip/lecture-notes/uni04/node15.html